themes/theme-ios11/pages/Search/components/Content/index.jsx

```diff
@@ -38,16 +44,19 @@ class SearchContent extends Component {
    */
   render() {
     const {
-      searchPhrase, showFilterBar, showNoResults,
+      searchPhrase, showFilterBar, showNoResults, pattern,
     } = this.props;
 
     return (
       <RouteContext.Consumer>
         {({ state, query, id: routeId }) => (
           <Fragment>
-            <BackBar title={searchPhrase} shadow={!showFilterBar} />
+            <BackBar
+              title={pattern === CATEGORY_ALL_PATTERN ? state.categoryName : searchPhrase}
```

Comment:
there are now console errors because `title` is sometimes `undefined` (especially when you reload a /all category)

Reply:
i fixed it